features-json/css-media-range-syntax.json

```diff
@@ -0,0 +1,506 @@
+{
+  "title":"media-query-range-syntax",
+  "description":"The Media Queries Level 4 specification includes some syntax improvements to make media queries using features that have a 'range' type, for example width or height, less verbose. Level 4 adds a range context for writing such queries.",
```

Comment:
> Level 4 adds a range context for writing such queries

Change to:

Can be used ordinary mathematical comparison operators: `>`, '<', `>=`, or '<='. For example:

Without this feature: `@media (min-width: 100px) and (max-width: 1900px)`
With this feature: `@media (100px <= width <= 1900px)`

Reply:
It is best if we can show the user how this syntax is used at a glance in the description.

Reply:
Strongly Agree